pkg/asset/rhcos/bootstrap_image.go

```diff
@@ -37,6 +39,13 @@ func (i *BootstrapImage) Generate(p asset.Parents) error {
 	p.Get(ic)
 	config := ic.Config
 
+	// Check for RHCOS image URL override
+	if boi := config.Platform.BareMetal.BootstrapMachineOSImage; boi != "" {
+		logrus.Info("Found override for Bootstrap OS Image.")
```

Comment:
If we're going to log something here it'd be nice to include the image override that's actually being used, just saying it's being overridden won't help us figure out when someone has this config messed up :)

Reply:
This is an older commit. Will update the new one.